config/default.yml

```diff
@@ -1138,6 +1138,13 @@ Layout/MultilineMethodDefinitionBraceLayout:
     - new_line
     - same_line
 
+Layout/MultilineMethodParameterLineBreaks:
+  Description: >-
+                 Checks that each parameter in a multi-line method definition
+                 starts on a separate line.
+  Enabled: false
+  VersionAdded: '1.31'
```

Comment:
`VersionAdded` is decided mechanically when a new release is cut. So it'd be better to replace with `<<next>>` token instead of fixed version.

https://github.com/rubocop/rubocop/blob/a5de777eb80f45d58564cacb6be4aa60072a5fa3/tasks/cut_release.rake#L32-L37